xls/dev_tools/make-compilation-db.sh

```diff
@@ -13,24 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -u
-set -e
-
-readonly OUTPUT_BASE="$(bazel info output_base)"
-
-readonly COMPDB_SCRIPT="${OUTPUT_BASE}/external/com_grail_bazel_compdb/generate.py"
-[ -r "${COMPDB_SCRIPT}" ] || bazel fetch @com_grail_bazel_compdb//...
-
-python3 "${COMPDB_SCRIPT}"
-
-# Massage the output so that clang-tidy fully undestands the compile commands:
-# remove flags where it gets confused.
-# Also, make sure that the command also contains -xc++ (bazel sometimes does
-# not add that in libraries that don't have a *.cc file but only *.h or *.inc)
-sed -i compile_commands.json -f - <<EOF
-s/"command": "\([^ ]*\) /"command": "\1 -x c++ /  # -xc++ as first argument.
-s/ -f[^ ]*/ /g             # remove all -fxyz options
-s/ --target[^ ]*/ /g       # target platform not needed, might confuse
-s/ -stdlib=libc++/ /       # otherwise, clang-tidy does not find c++ headers
-EOF
-
+echo "Sorry, currently no compilation DB with bazel 7 and bzlmod. Stay tuned."
```

Comment:
Maybe include a pointer to a tracking issue.

Reply:
I don't have a tracking issue, but I have a CL that fixes this (and also provides build_cleaner functionality) ready to go as soon as this is in.